api/config/v1alpha1/envoyproxy_types.go

```diff
@@ -105,25 +102,22 @@ type EnvoyProxyKubernetesProvider struct {
 	EnvoyService *KubernetesServiceSpec `json:"envoyService,omitempty"`
 }
 
-// ProxyLogging defines logging parameters for managed proxies. This type is not
-// implemented until https://github.com/envoyproxy/gateway/issues/280 is fixed.
+// ProxyLogging defines logging parameters for managed proxies.
 type ProxyLogging struct {
 	// Level is a map of logging level per component, where the component is the key
-	// and the log level is the value. If unspecified, defaults to "System: Info".
+	// and the log level is the value. If unspecified, defaults to "system: warn".
 	//
-	// +kubebuilder:default={system: info}
+	// +kubebuilder:default={system: warn}
 	Level map[LogComponent]LogLevel `json:"level,omitempty"`
 }
 
 // LogComponent defines a component that supports a configured logging level.
-// This type is not implemented until https://github.com/envoyproxy/gateway/issues/280
-// is fixed.
 // +kubebuilder:validation:Enum=system;upstream;http;connection;admin;client;filter;main;router;runtime
 type LogComponent string
 
 const (
-	// LogComponentSystem defines the "system"-wide logging component. When specified,
-	// all other logging components are ignored.
+	// LogComponentSystem defines the "system"-wide logging component.
+	// See more details: https://www.envoyproxy.io/docs/envoy/latest/operations/cli#cmdoption-l
 	LogComponentSystem LogComponent = "system"
```

Comment:
I dont see any component level called `system`https://github.com/envoyproxy/envoy/blob/cf1af7660d910c275bee1d7b34218c32c6419198//source/common/common/logger.h

Reply:
as we talked in WG meeting, will change this to `default`.